portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -247,6 +247,14 @@ definition {
 			expected = "${expectedPriorityValueList}");
 	}
 
+	macro assertPriorityFieldWithCorrectValue {
+		var actualPriorityDefaultValue = JSONUtil.getWithJSONPath("${responseToParse}", "$.priority");
```

Comment:
`assertPriorityFieldWithCorrectValue` and `assertPriorityFieldWithDefaultValue` do the same thing.
`assertPriorityFieldIsFilteredWithCorrectValue` and `assertPriorityFieldIsSortedWithCorrectValue` do the same thing. 

Let's think about how could we unify these methods